sites/platform/src/add-services/valkey.md

```diff
@@ -0,0 +1,644 @@
+you can use [default endpoint configuration](/create-apps/app-reference/single-runtime-image#relationships).
+
+With the above definition, the application container now has [access to the service](#use-in-app) via the relationship `<RELATIONSHIP_NAME>` and its corresponding [service environment variables](/development/variables/_index.md#service-environment-variables).
+
+{{< /codetabs >}}
+
+For PHP, enable the [extension](/languages/php/extensions) for the service:
+
+{{< codetabs >}}
+
++++
+title=Using default endpoints
++++
+
+```yaml {configFile="app"}
+applications:
+  # The name of the app container. Must be unique within a project.
+  <APP_NAME>:
+    # PHP extensions.
+    runtime:
+      extensions:
+        - valkey 8.0
+    # Relationships enable access from this app to a given service.
+    # The example below shows simplified configuration leveraging a default service
+    # (identified from the relationship name) and a default endpoint.
+    # See the Application reference for all options for defining relationships and endpoints.
+    relationships:
+      <SERVICE_NAME>:
+```
+
+<--->
+
++++
+title=Using explicit endpoints
++++
+
+```yaml {configFile="app"}
+applications:
+  # The name of the app container. Must be unique within a project.
+  <APP_NAME>:
+    # PHP extensions.
+    runtime:
+      extensions:
+          - valkey 8.0
+    # Relationships enable access from this app to a given service.
+    # The example below shows configuration with an explicitly set service name and endpoint.
+    # See the Application reference for all options for defining relationships and endpoints.
+    relationships:
+      <RELATIONSHIP_NAME>:
+        service: <SERVICE_NAME>
+        endpoint: valkey
+```
+
+{{< /codetabs >}}
+
+### Configuration example
+
+#### [Service definition](/add-services/_index.md)
+
+```yaml {configFile="services"}
+# The name of the service container. Must be unique within a project.
+valkey:
+  type: valkey-persistent:8.0
+  disk: 256
+```
+
+#### [App configuration](/create-apps/_index.md)
+
+{{< codetabs >}}
+
++++
+title=Using default endpoints
++++
+
+```yaml {configFile="app"}
+relationships:
+  valkey:
+```
+
+<--->
+
++++
+title=Using explicit endpoints
++++
+
+```yaml {configFile="app"}
+relationships:
+  valkey:
+    service: valkey
+    endpoint: valkey
+```
+
+{{< /codetabs >}}
+
+## Ephemeral Valkey
+
+By default, Valkey is an ephemeral service that serves as a non-persistent cache.
+Ephemeral Valkey stores data only in memory and requires no disk space.
+When the service reaches its memory limit, it triggers a cache cleanup.
+To customize those cache cleanups, set up an [eviction policy](#eviction-policy).
+
+Make sure your app doesn't rely on ephemeral Vedis for persistent storage as it can cause issues. For example, if a container is moved during region maintenance,the `deploy` and `post_deploy` hooks don't run and an app that treats the cache as permanent shows errors.
```

Comment:
```suggestion
Make sure your app doesn't rely on ephemeral Valkey for persistent storage as it can cause issues. For example, if a container is moved during region maintenance,the `deploy` and `post_deploy` hooks don't run and an app that treats the cache as permanent shows errors.
```